README.md

```diff
@@ -17,9 +17,39 @@ Then alter your homeserver configuration, adding to your `modules` configuration
 modules:
   - module: threepid_checker.ThreepidChecker
     config:
-      # TODO: Complete this section with an example for your module
+      # The URL to send requests to when checking if a 3PID can be associated to an account.
+      # See below for more information.
+      # Required.
+      url: https://foo/bar
+
+      # If true, the module will only send requests to the URL as part of registering a
+      # new user. This means no request will be sent to the URL when associating a 3PID with
+      # an already existing account.
+      # Optional, defaults to false.
+      only_check_at_registration: false
 ```
 
+The configured URL will be hit by a `GET` HTTP request, with 2 parameters to qualify the 3PID:
```

Comment:
I assume these are query parameters?

A downside of query parameters is that they'll leak into logs, but I'm guessing this is fitting an existing API?

Reply:
> I assume these are query parameters?

Yup.

> I'm guessing this is fitting an existing API?

Yep, it's designed to match https://github.com/matrix-org/sydent/blob/dinsic/docs/info.md#get-_matrixidentityapiv1internal-info

> A downside of query parameters is that they'll leak into logs

That's a known issue, and is tracked here: https://github.com/matrix-org/matrix-dinsic/issues/785. We'll have to figure out a migration to a POST endpoint at some point.